### PR TITLE
Support for mkisofs on Mac OS X

### PR DESCRIPTION
**Pull request by tweemeterjop**
Add support for Mac OS X by adding the correct path for vboxmanage and ignoring some specific warnings.

### CODE REVIEW

File: vmcloak/abstract.py [hunk 1]
```diff
@@ -21,9 +22,10 @@
 
 log = logging.getLogger(__name__)
 
-GENISOIMAGE_ERROR = (
-    "Warning: creating filesystem that does not conform to ISO-9660.\n"
-)
+GENISOIMAGE_WARNINGS = [
+    "Warning: creating filesystem that does not conform to ISO-9660.",
+    "Warning: creating filesystem that does not conform to ISO-9660. Warning: creating filesystem with (nonstandard) Joliet extensions but without (standard) Rock Ridge extensions. It is highly recommended to add Rock Ridge",
```
Comment (jbremer) [inline vmcloak/abstract.py]: Can you split this line over multiple lines so to adhere to pep8 80-char convention etc?

File: vmcloak/abstract.py [hunk 2]
```diff
@@ -263,11 +265,13 @@ def buildiso(self, mount, newiso, bootstrap, tmp_dir=None):
...
         ] + self.genisoargs + [outdir]
 
+        log.debug("Executing genisoimage: %s", " ".join(args))
         p = subprocess.Popen(
             args, stdout=subprocess.PIPE, stderr=subprocess.PIPE
         )
         out, err = p.communicate()
-        if p.wait() or out or err != GENISOIMAGE_ERROR:
+
+        if p.wait() or out or re.sub("[\s]+", " ", err).strip() not in GENISOIMAGE_WARNINGS:
```
Comment (jbremer) [inline vmcloak/abstract.py]: Same here, probably the easiest to put `warning = re.sub("[\s]+", " ", err).strip()` on the line above to make space.

Comment (jbremer) [inline vmcloak/abstract.py]: Also use double backslash for non-raw strings, i.e., `"[\\s]+"`.

**Comment by tweemeterjop** [comment]
Better now? :)

**Comment by jbremer** [comment]
Yes sir, thanks!

**Pull request merged**